ext_emconf.php

```diff
@@ -13,7 +13,7 @@
         'depends' => [
             'php' => '8.1.0-8.9.99',
             'typo3' => '12.4.0-12.4.99',
-            'rte_ckeditor' => '12.4.0-12.4.99',
+            'cms_rte_ckeditor' => '12.4.0-12.4.99',
```

Comment:
The extension name is still `rte_ckeditor`, so this looks wrong for me.  
It is shown in the composer.json in extras -> typo3/cms -> extension-key

Reply:
This change was to solve

https://github.com/netresearch/t3x-rte_ckeditor_image/actions/runs/8007170786/job/21870624384#step:5:38

Netresearch\RteCKEditorImage\Tests\Functional\DataHandling\RteImageSoftReferenceParserTest::updateReferenceIndexAddsIndexEntryForImage
TYPO3\TestingFramework\Core\Exception: Package "rte_ckeditor_image" depends on package "rte_ckeditor" which does not exist.

Reply:
well, then the test expectation is wrong.
The correct name should be assigned when the repository is checked out, but it seems that is working, just the runTests.sh doesn't has or get the correct name.

Reply:
> The extension name is still rte_ckeditor, so this looks wrong for me.

mhm ... for me it seems right:

public/typo3/sysext/core/ -> "cms-core"

so it should:

public/typo3/sysext/rte_ckeditor/ -> cms-rte-ckeditor

no?

also see:

https://github.com/netresearch/t3x-rte_ckeditor_image/blob/fix_build_errors/composer.json#L15
or
https://github.com/TYPO3-CMS/rte_ckeditor/blob/main/composer.json#L3

or, let me ask: do you see any problem?

Reply:
> or, let me ask: do you see any problem?

I suppose there aren't any problems in composer  but maybe in legacy installations.  
AFAIK In composer installations the file is not used anymore and you could write anything there.  
composer relies on the composer-names, so that's completely different, but in legacy installations folders are also identified based on the extension key, so at one end or another it will throw errors.

Reply:
runTests.sh needs certainly an adjustment, I think these lines never make sense for rte_ckeditor, perhaps there is more:
```
# Set $1 to first mass argument, this is the optional test file or test directory to execute
shift $((OPTIND - 1))
if [ -n "${1}" ]; then
    TEST_FILE="Web/typo3conf/ext/enetcache/${1}"
fi
```

Reply:
@DavidBruchmann, you are right.

the whole testing framework needs to be updated/revised to latest available version and newest best practices.
But IMO this is out of scope for this MR.

Reply:
EXT:news has a runTests.sh too ...

Reply:
Hi Together,

I resolved the discussion to be able to merge this PR and I don't see any required changes so far. If the are still open topics left pleas open a new issue. 

Thanks.